conans/client/migrations.py

```diff
@@ -73,7 +73,7 @@ def _make_migrations(self, old_version):
         version: ["4.1", "4.4", "4.5", "4.6", "4.7", "4.8", "4.9",
                   "5", "5.1", "5.2", "5.3", "5.4",
                   "6", "6.1", "6.2", "6.3", "6.4",
-                  "7", "7.1", "7.2"]
+                  "7", "7.1", "7.2", "7.3"]
```

Comment:
Uhm, yes, you are right, this wont migrate because the above comparison is done with "1.0" version. But I think this 7.3 shouldn't be added here, because this is intended to be the previous version. If it was unmodified from the default settings, then the migration to 1.0 won't warn about the backup.

Reply:
Yes, this has to be removed, please remove it.